Services/WalletConnectorService/Package.swift

```diff
@@ -13,7 +13,7 @@ let package = Package(
     ],
     dependencies: [
         .package(name: "Primitives", path: "../../Packages/Primitives"),
-        .package(url: "https://github.com/reown-com/reown-swift", exact: Version(stringLiteral: "1.6.0")),
+        .package(url: "https://github.com/gemwalletcom/reown-swift.git", branch: "remove-appkit"),
```

Comment:
```suggestion
        .package(url: "https://github.com/gemwalletcom/reown-swift.git", branch: "developer"),
```

let's keep it on `develop`